src/journal/journald-console.c

```diff
@@ -72,9 +72,9 @@ void server_forward_console(
         /* First: timestamp */
         if (prefix_timestamp()) {
                 assert_se(clock_gettime(CLOCK_MONOTONIC, &ts) == 0);
-                xsprintf(tbuf, "[%5"PRI_TIME".%06ld] ",
+                xsprintf(tbuf, "[%5"PRI_TIME".%06lu] ",
                          ts.tv_sec,
-                         ts.tv_nsec / 1000);
+                         (unsigned long)ts.tv_nsec / 1000);
```

Comment:
Please see the comment above from @karcherm. Dividing by ```1000LU``` will promote the result to ```long long``` and the build fails again:

```
In file included from ./src/basic/alloc-util.h:27:0,
                 from src/journal/journald-console.c:24:
src/journal/journald-console.c: In function 'server_forward_console':
src/journal/journald-console.c:75:32: error: format '%lu' expects argument of type 'long unsigned int', but argument 5 has type '__syscall_slong_t {aka long long int}' [-Werror=format=]
                 xsprintf(tbuf, "[%5"PRI_TIME".%06lu] ",
                                ^
./src/basic/macro.h:45:44: note: in definition of macro '_unlikely_'
 #define _unlikely_(x) (__builtin_expect(!!(x),0))
                                            ^
./src/basic/stdio-util.h:30:9: note: in expansion of macro 'assert_message_se'
         assert_message_se((size_t) snprintf(buf, ELEMENTSOF(buf), fmt, __VA_ARGS__) < ELEMENTSOF(buf), "xsprintf: " #buf "[] must be big enough")
         ^~~~~~~~~~~~~~~~~
src/journal/journald-console.c:75:17: note: in expansion of macro 'xsprintf'
                 xsprintf(tbuf, "[%5"PRI_TIME".%06lu] ",
                 ^~~~~~~~
cc1: some warnings being treated as errors
```